components/unit/consent-form/UnitConsentForm.vue

```diff
@@ -91,7 +91,7 @@ import { createObjectURL, mimeTypes } from '@/utils/utils'
 
 // In the case of changes that would break the import, this version number must be incremented
 // and the function versionCompatibilityHandler of import.vue must be able to handle previous versions.
-const VERSION = 3
+// const VERSION = 3
```

Comment:
I guess we can erase this

Reply:
If you delete the if in versionCompatibilityHandler() I think so too